### PR TITLE
[coin-tester] Polkadot support

### PR DESCRIPTION
**Pull request by lvndry**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [x] **Covered by automatic tests.** 

### 📝 Description

Support coin-tester for polkadot

Screenshot 2024-05-30 at 21 24 28

### ❓ Context

- **JIRA or GitHub link**: https://ledgerhq.atlassian.net/browse/LIVE-11295

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

**Comment by pepoviola** [comment]
Hi @lvndry, I just checked running your config, with rococo-local and only alice (as validator). And all the dev accounts are there.
image

Did you manage to spawn the network ok? can you share the logs of the failed test? because the devs accounts should be available to make transfers.

Thx!

**Review by mcoursier-ledger — DISMISSED** [review]

File: libs/coin-modules/coin-polkadot/src/test/fixtures.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import { CryptoCurrency } from "@ledgerhq/types-cryptoassets";
+import { PolkadotAccount } from "../types";
+import { decodeAccountId } from "@ledgerhq/coin-framework/account";
+import { getDerivationScheme, runDerivationScheme } from "@ledgerhq/coin-framework/derivation";
+import { SubAccount } from "@ledgerhq/types-live";
+import BigNumber from "bignumber.js";
+
+export const makeAccount = (
```
Comment (hedi-edelbloute) [inline libs/coin-modules/coin-polkadot/src/test/fixtures.ts:8]: there is already a createFixtureAccount, maybe we can do something with it : ledger-live/libs/coin-modules/coin-polkadot/src/types/bridge.fixture.ts

**Review by hedi-edelbloute — APPROVED** [review]
lgtm

File: libs/coin-modules/coin-polkadot/src/test/fixtures.ts (hunk 1 above)
Comment (lvndry) [reply libs/coin-modules/coin-polkadot/src/test/fixtures.ts:8]: Agreed, the only issue is that there's some `faker` in the bridge fixture
We need something more deterministic for the coin tester

**Review by hedi-edelbloute — APPROVED** [review]

**Review by hedi-edelbloute — DISMISSED** [review]

**Pull request merged**